pkg/generate/app/sourcelookup.go

```diff
@@ -204,14 +204,17 @@ func (r *SourceRepository) Detect(d Detector, dockerStrategy bool) error {
 	if r.info != nil {
 		return nil
 	}
+
```

Comment:
Are the new lines here intentional? It's strange to change this file as part of this PR just for the empty lines.